src/lib/getPageType.ts

```diff
@@ -0,0 +1,66 @@
+import { Request } from "express"
+import { OwnerType, PageOwnerType } from "@artsy/cohesion"
+import { camelCase } from "lodash"
+
+export function getPageTypeFromReq(
+  req: Request
+): {
+  pageParts: string[]
+  pageType: PageOwnerType
+  pageSlug: string
+} {
+  const pageParts = req.path.split("/")
+  const pageSlug = pageParts[2]
+  const type = formatOwnerTypes(pageParts[1])
+  const pageType = OwnerType[type] || type
+
+  return {
+    pageParts,
+    pageSlug,
+    pageType,
+  }
+}
+
+export function getPageTypeFromClient(): {
```

Comment:
@eessex - a TODO i've had laying around for a while was to investigate using [`path-to-regex`](https://github.com/pillarjs/path-to-regexp) here to make things a little more bulletproof. 

We use it in a few places, for example [here](https://github.com/artsy/force/blob/master/src/desktop/analytics/main_layout.ts#L13-L42). Not sure if its totally appropriate, but worth exploring.

Reply:
Checked this out but couldn't find how it would bring us more safety in this use-case, the lib's parsing still returns an array on pathnames so while more abstracted it would be functionally equivalent to `req.path.split("/")`

Reply:
Cool cool 👍 was curious if there might be some feature that would benefit